lib/models/movie.js

```diff
@@ -7,7 +7,7 @@ module.exports = Bookshelf.Model.extend({
   serialize: function () {
     return {
       id: this.get('id'),
-      title: this.get('title'),
+      name: this.get('name'),
```

Comment:
Yeah, so you'll want to keep this as `title`, but get it from the `name` column!